demosplan/DemosPlanCoreBundle/Controller/Report/DemosPlanReportController.php

```diff
@@ -70,12 +70,12 @@ public function exportProcedureReportAction(
         NameGenerator $nameGenerator,
         PermissionsInterface $permissions,
         ProcedureHandler $procedureHandler,
-                              $procedureId
+        $procedureId
     ): Response {
         $slugify = new Slugify();
         $procedure = $procedureHandler->getProcedureWithCertainty($procedureId);
 
-        $currentTime = Carbon::now();
+        $currentTime = Carbon::now()->timezone('Europe/Berlin');
```

Comment:
I would not hard code 'Europe/Berlin', rather try to figure out how to get it from a system configuration., I wonder if this happens in prod (it can be only the test env that does not have Europe Berlin configured